adapters/repos/db/vector/hnsw/vector_cache_prefiller.go

```diff
@@ -123,13 +123,13 @@ func (pf *vectorCachePrefiller[T]) logLevel(level, count int, before time.Time)
 }
 
 func (pf *vectorCachePrefiller[T]) logTotal(count, limit int, before time.Time) {
-	pf.logger.WithFields(logrus.Fields{
+	/*pf.logger.WithFields(logrus.Fields{
```

Comment:
remove if we don't need it?

Reply:
Please don't remove this. This is very helpful for CS when doing manual upgrades, etc. to see when the vector cache is ready.

Reply:
It was spamming my logs when I was debugging, I'll put it back in